src/lib/operators/get_table.cpp

```diff
@@ -0,0 +1,15 @@
+#include <memory>
+#include <string>
+
+#include "get_table.hpp"
+#include "storage/storage_manager.hpp"
+
+namespace opossum {
+
+GetTable::GetTable(const std::string& name) : _table_name(name) {}
+
+const std::string& GetTable::table_name() const { return _table_name; }
+
+std::shared_ptr<const Table> GetTable::_on_execute() { return StorageManager::get().get_table(_table_name); }

```

Comment:
We could add an assert or debug assert to check that the table actually exists in the StorageManager. I can see that your implementation of the StorageManager uses `.at` to throw an error for an invalid table name. Adding an explicit assert or debug assert here could help improve visibility into the error cause and could speed up error resolution for calling code.